### DISCUSSION TITLE
Services defined only by thousands of ICD Codes

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#63] [Q&A] [answered]
I have a payer contract that identifies some services such as PTCA and Coronary Surgery by a list of ICD10 codes only.  Coronary Surgery's list is almost 4k codes long.  None of these will have a standard gross charge amount and the reimbursement is a case rate so I'm curious how these should be handled on the report? It seems excessive to list all 4k lines individually on the report when the only difference would be the code and it doesn't seem appropriate to list 4k code/codetypes on one line.  How do we make this easy?

**Comment by nmayle** [comment]
A question for clarification.  When multiple codes are listed on the same row/line, does that mean AND or OR?

It sounds like you want an OR: the same case rate charge applies to a large set of ICD10s.

My assumption has always been that when multiple codes are listed for the same row, that is an AND (of the codes) and if you want an OR (of the codes) you can always have multiple rows.

So, you would list the separate codes in separate rows; they just happen to have the same charge.

@CMS please clarify the semantics.

**Reply by JaSimps5** [reply]
For this particular scenario, it's an Or statement.   

However, there are other scenarios that have ICD10's AND other conditions, Just as an example a contract that pays differently for an Adult vs Adolescent Transplant procedure.  When that list of ICD10 codes is in the thousands and then you have to duplicate it to account for the AND scenarios, it makes these files extremely large if we are listing each code out on multiple rows.

**Reply by nmayle** [reply]
As one of the many consumers of these files, it is very important to be able to reliably interpret the data -- clear semantics.  If the multiple codes on a row can be interpreted as AND or OR, then how would we know which to use when?

Since multiple rows can be used for OR and there isn't another mechanism for AND, I would suggest that the only reasonable way to use multiple codes on the same row is AND.

True, this will result in your case (and others) in thousands of rows with almost exactly the same information, but, I strongly believe that the consumers of the files would rather have large files with a clear interpretation of each row than a smaller file that is ambiguous.  Many of us are also using Transparency in Coverage data files and those are often much much larger -- terabytes every month from a single payer.

**Reply by nmayle** [reply]
BTW: the same question comes up with the modifiers field -- it is an AND or an OR if there are multiple modifiers.

@CMS every column that can have multiple values or have multiple columns for the same field should have clearly documented semantics as to whether it is an OR or an AND.  Ambiguity undermines the goal of the whole effort of price transparency: the ability to interpret and compare prices.

**Comment by RidenourK** [comment, marked as answer]
> I have a payer contract that identifies some services such as PTCA and Coronary Surgery by a list of ICD10 codes only. Coronary Surgery's list is almost 4k codes long. None of these will have a standard gross charge amount and the reimbursement is a case rate so I'm curious how these should be handled on the report? It seems excessive to list all 4k lines individually on the report when the only difference would be the code and it doesn't seem appropriate to list 4k code/codetypes on one line. How do we make this easy?

@JaSimps5. Thank you for your question. 

Could you please provide a few sample rows of this MRF that include the codes in question or a sample of the codes with any corresponding standard charge information to the CMS HPT mailbox at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov)?

**Reply by JaSimps5** [reply]
Sent examples

**Reply by dataMuley** [reply]
Was this actually answered?

**Reply by RidenourK** [reply]
@DataMuley. Thank you for your question. 

The poster’s specific question about this unique contracting scenario was addressed via the CMS Price Transparency Hospital Charges mailbox.
